<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.0 (Python 3.12.2 on win32)" generated="2024-03-15T20:39:12.414178" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="c:\Users\nothing\.vscode\extensions\robocorp.robotframework-lsp-1.11.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2024-03-15T20:39:12.448729" elapsed="3.406533"/>
</kw>
<status status="PASS" start="2024-03-15T20:39:12.447730" elapsed="3.407532"/>
</test>
<status status="PASS" start="2024-03-15T20:39:12.417191" elapsed="3.439069"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" skip="0" id="s1" name="Robot Interactive Console">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2024-03-15T20:39:12.401169" level="WARN">Error in file 'c:\Users\nothing\.vscode\extensions\robocorp.robotframework-lsp-1.11.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
